modules/appeals_api/app/swagger/appeals_api/v1/schemas/notice_of_disagreements.rb

```diff
@@ -143,6 +154,22 @@ class NoticeOfDisagreements
                       key :description, 'value of submitted homeless key'
                     end
 
```

Comment:
I think these should have character lengths as well, right?

Reply:
Will do!